client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEditForm.tsx

```diff
@@ -114,25 +115,38 @@ export const RequestLineEditForm = ({
   const formatNumber = useFormatNumber();
   const { lines } = useRequest.line.list();
   const requestedQuantity = draftLine?.requestedQuantity ?? 0;
+  const originalItemName = lines?.find(({ item }) => item.id === item.id)
+    ?.itemName;
```

Comment:
sorry, I meant `item.id === item.id` doesn't make sense to me its always true.

How is this working in general? were is the item in the props coming from? and why are you using it to find a line from within the RequestLineEditForm? Shouldn't there already be a itemName in the current line?

Reply:
Ah I was trying to get the line item and prop item at the same time but can see how that is confusing and might fudge the code... prop item is the selected item

Reply:
Item from prop is the currently selected item and so I'm trying to get the item name from the line

Reply:
I can rename to currentItem ~